cime_config/tests.py

```diff
@@ -427,38 +427,45 @@
     },
 
     "e3sm_scream" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0"),
+    ),
+
+    "e3sm_scream_v0" : {
+        "time"    : "03:00:00",
+        "inherit" : ("e3sm_scream_v0_lowres"),
```

Comment:
I'm okay with the two levels of redirection here, it makes sense conceptually and for consistency, but we could simplify this if we wanted. I suppose maybe we eventually want to get rid of e3sm_scream and just call e3sm_scream_v0 directly?

Reply:
Yes, that's my thought. But since that also requires changing e3sm scripts, I wanted to first do this step.